### ISSUE TITLE
Some warnings due to updating PyTorch to version 0.4.0

### ISSUE THREAD

**Issue by morawi** [creafz/pytorch-cnn-finetune#5]
After updating PyTorch to version 0.4.0, a few warnings appeared iin the Cifar10 example.
1- UserWarning: invalid index of a 0-dim tensor. This will be an error in PyTorch 0.5. Use tensor.item() to convert a 0-dim tensor to a Python number
 test_loss += criterion(output, target).data[0]   
corrected to: test_loss += criterion(output, target).item()
 total_loss += loss.data[0]
corrected to: total_loss += loss.item()

2- UserWarning: volatile was removed and now has no effect. Use `with torch.no_grad():` instead.
  data, target = Variable(data, volatile=True), Variable(target)
Note sure how to correct this....  `with torch.no_grad():`

**Comment by creafz** [comment]
All PyTorch 0.4 warnings should be fixed in 73a0b57
